log/sockstatlog/logger_noop.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !tailscale_go || !(darwin || ios || android)
+
+// Keep build tags in sync with net/sockstats/sockstats_noop.go
```

Comment:
Rather than doing this (which requires manual upkeep and an extra file), I think it's OK to include the logger code regardless of build tags (since it will build with the standard Go too). 

We could instead have a `sockstats.IsAvailable` const that is set to true in `sockstats_tsgo.go` and false in `sockstats_noop.go`.

Reply:
done